web/src/components/profile/approved-researcher/TrainingCertificateError.tsx

```diff
@@ -0,0 +1,22 @@
+type TrainingCertificateErrorProps = {
+  text: string;
+};
+
+const manualApprovalEmailAddress = "ig-training@ucl.ac.uk";
+const manualApprovalEmailBody = encodeURI("Dear UCL Information Governance,\n\n ...");
+const manualApprovalEmailSubject = "Training certificate";
+
+export default function TrainingCertificateError(props: TrainingCertificateErrorProps) {
+  return (
+    <p>
+      {props.text}
+      {" If required, please email your certificate to "}
```

Comment:
```suggestion
      {" Oops! Something has gone wrong, please email your certificate to "}
```

I think the "if required" adds ambiguity, how do they know it's required or not? Just tell them to do it because they shouldn't be seeing the message otherwise, surely?

Reply:
Ahhh, in which case, maybe something like "Please double check your name on the certificate is correct and try again. If the problem persists, contact us at ..."